packages/frontend/src/assets/logos/dexes/swapr.tsx

```diff
@@ -0,0 +1,19 @@
+export function SwaprLogo(props: any) {
+    return (
+        <svg
+            width="21.621759"
+            height="21.62163"
+            viewBox="0 0 21.621759 21.62163"
+            fill="none"
+            version="1.1"
+            {...props}
+        >
+            <path
+                fill-rule="evenodd"
```

Comment:
Camel case props here.

Reply:
Addressed in 261709c46d2d09cc3688acc348b577ab5d969315